$MirOS: ports/print/gv/patches/patch-source_signal_c,v 1.1.7.1 2005/03/18 15:51:06 tg Exp $
--- source/signal.c.orig	Fri Apr 25 21:59:40 1997
+++ source/signal.c	Sun Apr 30 16:34:31 2006
@@ -41,7 +41,7 @@
 #endif
 
 #include "paths.h"
-#include INC_X11(Intrinsic.h)
+#include <X11/Intrinsic.h>
 
 
 #if defined(USE_SIGNAL_HANDLER)
@@ -59,7 +59,7 @@
 #include "d_memdebug.h"
 #include "main_resources.h"
 #include "main_globals.h"
-#include "signal.h"
+#include "gvsignal.h"
 
 #ifdef SIGNALRETURNSINT
 #   define SIGVAL int
